examples/03_connectivity/plot_probabilistic_atlas_extraction.py

```diff
@@ -23,16 +23,9 @@
 # Retrieve the atlas and the data
 from nilearn import datasets
 atlas = datasets.fetch_atlas_msdl()
-atlas_filename = atlas['maps']
-
-# Load the labels
-import numpy as np
-csv_filename = atlas['labels']
-
-# The recfromcsv function can load a csv file
-labels = np.recfromcsv(csv_filename)
-names = labels['name']
+atlas_filename, labels = atlas['maps'], atlas['labels']
```

Comment:
I would like these 2 assignments split in 2 different lines: it makes it easier to follow for the beginner.